### DISCUSSION TITLE
rich hander with rorating

### DISCUSSION THREAD

**Discussion by LCH-1** [Textualize/rich#2814] [Q&A]
Is it possible to use features like TimedRotatingFileHandler in rich, or conversely, is there a way to use the rich format in TimedRotatingFileHandler?

**Comment by bsridatta** [comment]
Can anyone answer this thanks? @willmcgugan  I am using the TimedRotatingFileHandler too, and I tail this in a separate terminal than the STDOUT, as I use rich to create a nice UI there and I do  not want any logs to be printed there. I am the logs to be formatted by rich. Anyway to do that? Thank you!

**Comment by scorpion35** [comment]
This would be awesome if it's supported. Think rich is mainly made for console apps?
